src/lib/Scrollback.rei

```diff
@@ -0,0 +1,11 @@
+type t;
+
+let push: (~cells: array(Vterm.ScreenCell.t), t) => t;
```

Comment:
If the API is mutable, it should return `unit` to indicate that. "Hybrid" APIs are inadvisable.

Reply:
Makes sense!